test/JDBC/expected/TestSpatialPoint-vu-prepare.out

```diff
@@ -175,7 +175,7 @@ prepst#!#exec#!#GEOMETRY|-|location|-|LINESTRING(1 2, 3 4):4326
 ~~ERROR (Message: Unsupported geometry type)~~
```

Comment:
Let's add the test case given in issue report as well.

Reply:
Done